celery/backends/base.py

```diff
@@ -256,6 +256,19 @@ def fail_from_current_stack(self, task_id, exc=None):
             self.mark_as_failure(task_id, exc, exception_info.traceback)
             return exception_info
         finally:
+            if sys.version_info >= (3, 4, 0):
```

Comment:
3.5?

Reply:
This property is from 3.4 and above.  ok  it will change this to 3.5